bin/config.js

```diff
@@ -1,19 +1,11 @@
-const rc = require('rc')
+const fsproxy = require('fs-proxy')
 
-module.exports = rc('cordlr', {
```

Comment:
Yeah sure, that sounds good.

Btw, for the reading, you can use `require(absolutePath)` just to skip `JSON.parse` and stuff.